cmd/topology/main.go

```diff
@@ -14,10 +14,12 @@ import (
 	"strings"
```

Comment:
Update Copyright year on line 3 and on other files that were modified

Reply:
fixed